visual-recognition/src/main/java/com/ibm/watson/developer_cloud/visual_recognition/v3/model/VisualCollectionImages.java

```diff
@@ -0,0 +1,34 @@
+package com.ibm.watson.developer_cloud.visual_recognition.v3.model;
+
+import java.util.Date;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+public class VisualCollectionImages extends GenericModel {
+	/**
+	 * VisualCollectionImages.
+	 */
+	@SerializedName("image_id")
+	private String image_id;
+	private Date created;
+	private String image_file;

```

Comment:
Use CanelCase instead of snake_case for variables and methods.

See other models for reference